cmd/initialize/prepare.go

```diff
@@ -0,0 +1,74 @@
+package initialize
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"strings"
+
+	"numerous.com/cli/cmd/initialize/wizard"
+	"numerous.com/cli/cmd/output"
+	"numerous.com/cli/internal/dir"
+	"numerous.com/cli/internal/manifest"
+	"numerous.com/cli/internal/python"
+)
+
+func PrepareInit(args []string) (string, *manifest.Manifest, error) {
+	appDir, err := os.Getwd()
+	if err != nil {
+		slog.Info("An error occurred when trying to get the current user path during init process.", slog.String("error", err.Error()))
+		fmt.Println(err)
```

Comment:
Are we displaying the error twice?

Reply:
And should we instead be using `output.PrintErrorDetails`

Reply:
Removed `slog.Info` and used `output.PrintErrorDetails` on the outer level, instead of this `fmt.Println`